src/app/dashboard/service/api/dailyRecordService.ts

```diff
@@ -383,37 +383,37 @@ export class DailyRecordService {
     }
 
     /**
-     * 
-     * @param hiveId 
-     * @param range 
+     *
+     * @param hiveId
+     * @param range
      */
     public getTempIntMaxByHive(hiveId: string, range: Date[]): Observable<any[]> {
         return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/tMax/' + hiveId, range).map(_elt => _elt.map(_value => {
             return { date: _value.date, value: this.unitService.convertTempFromUsePref(_value.value, this.unitSystem), sensorRef: _value.sensorRef};
         }));
     }
     /**
-     * 
-     * @param hiveId 
-     * @param range 
+     *
+     * @param hiveId
+     * @param range
      */
     public getHintByHive(hiveId: string, range: Date[]): Observable<any> {
         return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/hInt/' + hiveId, range);
     }
 
     /**
-     * 
-     * @param hiveId 
-     * @param range 
+     *
+     * @param hiveId
+     * @param range
      */
     public getBroodByHive(hiveId: string, range: Date[]): Observable<any[]> {
-        return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/brood/' + hiveId, range);
+        return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/brood/old/' + hiveId, range);
```

Comment:
What does this /old/ mean?

Reply:
server response 404 for this request 'dailyRecordsTH/brood/'.  this service REST URL is not found in back end. I checked your last deployed version 'https://bzz.mellisphera.com/', it use that  'dailyRecordsTH/brood/old/' URL. it is working properly. I think small code issue  from last version.

Reply:
That is weird, 
looking to the database the *dailyRecordsTH* collection has a *brood* item that seems the target of this request

![Capture d’écran 2020-07-21 à 10 36 34](https://user-images.githubusercontent.com/41069288/88032343-1d683180-cb3e-11ea-8158-c530717f2e77.png)

we need to understand further why it appears this way.